widgetastic_manageiq/__init__.py

```diff
@@ -4951,6 +4955,48 @@ def schedule_migration(self, plan_name, cancel=False, after_mins=1):
 
         # TODO: Create new method to unschedule the migration plans.
 
+    def schedule_cutover(self, plan_name, cancel=False, after_mins=5):
+        try:
+            el = self._get_plan_element(plan_name)
+            cutover_button = self.browser.element(self.ITEM_CUTOVER_BUTTON_LOCATOR, parent=el)
+            wait_for(
+                lambda: cutover_button.is_enabled(),
+                delay=5,
+                num_sec=120,
+                message="waiting for cutover button to be enabled.",
+            )
+            cutover_button.click()
+            if cutover_button.text == "Schedule" and not cancel:
```

Comment:
This doesn't look right . The cutover_button is different from schedule button right ?

You can add one more parameter start_migration_immediately=True.By default select_date_and_time radio button is selected.

def schedule_cutover(self, plan_name, cancel=False, after_mins=5, start_migration_immediately=True)

You just have to check 
    
    if not cancel : 
          if start_migration_immediately:
               # By default current date and time are selected
                start_migration_immediately.click()
          schedule.click()